html-relative-links/about-me/about-me.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>About Me</title>
+  <style>
+    img {
+      width: 300px;
+    }
+  </style>
+</head>
+<body>
+  <ul>
+    <li><a href="../index.html">Home Page</a></li>
+    <li><a href="../about-me/about-me.html">About Me</a></li>
```

Comment:
Since this file is already inside the `about-me` folder you don't need to leave the folder then go back into it. You can just use the filename here: `about-me.html`